cni/pkg/iptables/testdata/default.golden

```diff
@@ -0,0 +1,18 @@
+iptables -t mangle -N ISTIO_PRERT
```

Comment:
For followup: Would be nice to get architecture docs around what this does and why

Reply:
This file? Or this `iptables` line?

Reply:
Yeah I meant the whole file; tbh, I don't love how little our iptables logic is documented

Reply:
For the file - in the unit tests, we redirect the output of the raw `iptables` commands to a series of ordered strings, we compare that to this golden file 1-to-1 - this `.golden` is the the `expected` result, versus the `actual` result from running the iptables builder to generate inpod capture rules.

So this `.golden` is the complete set of iptables rules+entries you will see in-pod, and if it isn't, because you changed the rules without changing this, the test will fail.

It's been a very helpful correctness check internally, as slight changes or misses in the actual, "real" iptables rulesets we generate can otherwise be easy to miss.

Reply:
> tbh, I don't love how little our iptables logic is documented

I agree, it's very fiddly to boot - the testing setup here helps control for some of that, and I think in general we can share a lot more iptables stuff with sidecar now, and document the result better. Adding a note here: https://github.com/istio/istio/issues/49007